subsys/net/l2/openthread/openthread.c

```diff
@@ -265,6 +265,12 @@ static enum net_verdict openthread_recv(struct net_if *iface,
 	recv_frame.mInfo.mRxInfo.mLqi = net_pkt_ieee802154_lqi(pkt);
 	recv_frame.mInfo.mRxInfo.mRssi = net_pkt_ieee802154_rssi(pkt);
 
+#if defined(CONFIG_NET_PKT_TIMESTAMP)
+	struct net_ptp_time *time = net_pkt_timestamp(pkt);
+
+	recv_frame.mInfo.mRxInfo.mTimestamp = time->second * USEC_PER_SEC +
```

Comment:
What is the RX timestamp used in OT?

Reply:
There are several places:
* there's an optional time synchronization service in OT,
* NCP encapsulates this value and sends to host, I'm not exactly sure what does the host do with it, maybe just some stats,
* and more importantly, it'll be needed in one of the Thread 1.2 features - 802.15.4 CSL (coordinated sample listening).

We need the timestamp value for Zigbee as well.